[Upgrade to 1.6.6] mapcache.xml is missing

After upgrading to `1.6.6`, the file `apache/mapcache.xml` is missing and broke the build...

````
AH00526: Syntax error on line 51 of /var/www/vhosts/nyon-geoportail/private/dev/apache/tiles.conf:
failed to parse file /var/www/vhosts/nyon-geoportail/private/dev/apache/mapcache.xml. Is it valid XML?
Action 'graceful' failed.
The Apache error log may have more information.
CONST_Makefile:781: recipe for target '.build/apache.timestamp' failed
make: *** [.build/apache.timestamp] Error 1
Traceback (most recent call last):
  File ".build/venv/bin/c2ctool", line 9, in <module>
    load_entry_point('c2cgeoportal==1.6.6', 'console_scripts', 'c2ctool')()
  File "/var/www/vhosts/nyon-geoportail/private/dev/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 105, in main
    c2ctool.upgrade()
  File "/var/www/vhosts/nyon-geoportail/private/dev/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 241, in upgrade
    self.step2()
  File "/var/www/vhosts/nyon-geoportail/private/dev/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 394, in step2
    check_call(["make", "-f", self.options.file, "build"])
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', '-f', 'instance_dev.mk', 'build']' returned non-zero exit status 2
````

It's probably related to #1720

--- a/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
+++ b/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
@@ -758,7 +758,7 @@ print/WEB-INF/lib/postgresql-9.3-1102.jdbc41.jar:
 apache/mapcache.xml: tilegeneration/config.yaml .build/dev-requirements.timestamp
 	$(VENV_BIN)/generate_controller --generate-mapcache-config
 
-apache/mapcache.xml: tilegeneration/config.yaml .build/dev-requirements.timestamp
+apache/tiles.conf: tilegeneration/config.yaml .build/dev-requirements.timestamp
 	$(VENV_BIN)/generate_controller --generate-apache-config
 
 # Apache config

